package.nix

```diff
@@ -214,6 +215,8 @@ in {
     man # for testing `nix-* --help`
   ] ++ lib.optionals (doInstallCheck || enableManual) [
     jq # Also for custom mdBook preprocessor.
+  ] ++ lib.optionals (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
+    mesonEmulatorHook
```

Comment:
This should also go in the `package.nix` for the test exe itself, btw.

Reply:
Done, see 1e2affa